packages/wonder-blocks-icon/docs.md

```diff
@@ -0,0 +1,135 @@
+All of the `small` icons by name:
+
+```jsx
```

Comment:
These first two examples could be combined into a single one that uses a table with the first column being the names of the icons and the second and third being the small and medium icons.

Reply:
I believe there's a way to disable the "editor".  I think we should do that for this one since the code isn't very usefully to look at from a documentation point of view.

Reply:
I consolidated the two examples, but I don't think I'm going to put any more time into figuring out how to disable the code preview, since it seems like a pretty minor nice-to-have.